critiquebrainz/frontend/templates/macros.html

```diff
@@ -53,6 +53,13 @@
         {{ _('Artist') }}
       </span>
     {% endif %}
+  {% elif entity_type == 'label'  %}
+    <img src="{{ get_static_path('images/placeholder_place.svg') }}" {{ attributes }} />
```

Comment:
`placeholder_disc.svg` would be much relating to labels, imho.

Reply:
Updated.